openml-caret/README.md

```diff
@@ -0,0 +1,21 @@
+OpenML Provider for caret models
+------------------------------------------------------------
+
+This module contains an OpenML provider for loading custom code that implements
+Feedzai's caret API.
+
+The implemented "Classifier" should define the following methods: 
+
+```caret
+# Please add necessary components here.
```

Comment:
you can copy the required methods from https://github.com/feedzai/feedzai-openml-r/blob/master/openml-generic-r/README.md